src/stacks/web-crawler-stack.ts

```diff
@@ -67,10 +70,11 @@ export class WebCrawlerStack extends Stack {
       createContextTablePolicy,
       kendra: props.kendra,
       historyTable,
+      workingBucket,
       webCrawlerStateMachineArn,
     });
 
     // Create the state machine
-    new WebCrawlerStateMachine(this, 'WebCrawlerStateMachine', { steps });
+    const webCrawlerStateMachine = new WebCrawlerStateMachine(this, 'WebCrawlerStateMachine', { steps, workingBucket, webCrawlerStateMachineArn});
```

Comment:
Nit: I think we can delete the variable as it's unused :)

Reply:
Fixed.